rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -478,6 +479,33 @@ rudder.server.relay.api=https://localhost/rudder/relay-api
 
 "
 
+  # - 4.1.0 : Add property to define hooks ignore suffixes
+  STEP="Add property to define the list of suffixes to ignore hooks"
+
+  check_and_add_config_property rudder.hooks.ignore-suffixes "
+####################
+# Server side Hooks #############################################################
+####################
+
+# This property contains the comma separated list of extension that will be check 
+# before running a hook under /opt/rudder/etc/hooks.d. 
+# If an executable file has one of the following extensions, it
+# will be IGNORED and the corresponding hook skipped. Non executable files are 
+# always ignored, with or without any of these extensions. 
```

Comment:
Can we please consistently use "suffixes" instead of "extensions"? Or if you think it should be "extension", then rename the property, but don't mix and match.